.swiftlint.yml

```diff
@@ -0,0 +1,64 @@
+# rule identifiers to exclude from running
+disabled_rules:
+  - trailing_whitespace
+  - todo
+  - conditional_binding_cascade
+  - mark
+  - vertical_whitespace
+  - syntactic_sugar
+  - unused_closure_parameter
+  - closure_parameter_position
+  - redundant_string_enum_value
+  - large_tuple
+  - shorthand_operator
+  - vertical_parameter_alignment
+  - identifier_name  
+
+# some rules are only opt-in
+opt_in_rules:  
+  - force_unwrapping
+  - private_action
+  - explicit_init
+
+# paths to include during linting. `--path` is ignored if present.
+included:
+  - Riot
+
+# paths to ignore during linting. Takes precedence over `included`.
+excluded:
+  - Carthage
+  - Pods
+  - Riot/Generated/  
+
+line_length:
+  warning: 250
```

Comment:
250 seems huge. Error on 160 seems more reasonable.

Reply:
Yes it seems huge but several delegate methods and SwiftGen constants are near 200 characters

Reply:
`several delegate methods`
This is another reason I hate them but well :/